solutions/Basic-1-R.txt

```diff
@@ -0,0 +1,11 @@
+[1] "There are  26  sites."
+
+[1] "30  birds were counted at the 7th site."
+
+[1] "32  birds were counted at the last site."
+
+[1] "A total of  955  birds were counted across all sites."
+
+[1] "The average number of birds is  37  per site."
+
+[1] "A total of  248  birds were counted at sites with codes beginning with C."
```

Comment:
Let's skip the sentences here and just print out the numbers representing the actual results.